cmd/backup/main.go

```diff
@@ -517,6 +535,18 @@ func (s *script) copyBackup() error {
 		s.logger.Infof("Uploaded a copy of backup `%s` to bucket `%s`.", s.file, s.c.AwsS3BucketName)
 	}
 
+	// WebDAV file upload
+	if s.webdavClient != nil {
+		bytes, err := os.ReadFile(s.file)
+		if err != nil {
+			return fmt.Errorf("copyBackup: error reading the file to be uploaded: %w", err)
+		}
+		if err := s.webdavClient.Write(filepath.Join(s.c.WebdavPath, name), bytes, 0644); err != nil {
```

Comment:
I think this might be a bug: using `filepath.Join` will strip the double forward slash following the protocol: https://go.dev/play/p/C9Hx4i92jEn

Alternatively you could use a `net/url.URL` (https://pkg.go.dev/net/url#URL) or just do `fmt.Sprintf` I guess?

Reply:
Sorry, this only applies to the logging below, I was confused by the missing slash in the log output.

Reply:
Ok I see the case!  
But:  
I didnt find any "join" for `net/url.URL`.
And if I use `fmt.Sprintf("%s%s%s", s.c.WebdavUrl, s.c.WebdavPath, name)` there will be different results:

- `WEBDAV_PATH: /my/directory/` vs `WEBDAV_PATH: /my/directory`

Maybe something like [this](https://go.dev/play/p/aOOYJTTYZVo)?

Reply:
Since this is just the log output we could also get away with phrasing it differently (separating the URL and the path) which frees us from having to deal with these things, but your proposal is also great. I'm fine with both.